### PR TITLE
Update mongodb-connector-reference.adoc

### PR DESCRIPTION
**Pull request by george-hoffer**
Tested in local build and good to go.

### CODE REVIEW

File: modules/ROOT/pages/mongodb/mongodb-connector-reference.adoc [hunk 1]
```diff
@@ -14,2281 +18,2417 @@ MongoDB is an open source, high-performance, schema-free, document-oriented data
...
 
 
 [[createFileFromPayload]]
-== Create File From Payload
+=== Create File From Payload
 `<mongo:create-file-from-payload>`
 
 +++
-Creates a new MuleGridFSFile in the database, saving the given content, filename, contentType, and extraData, and answers it.
+Creates a new MuleGridFSFile in the database, saving the given content, filename, contentType, and extraData, and answers. it.
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/mongodb/mongodb-connector-reference.adoc]: I don't understand this. Answers what? Maybe it should be "answers in it." meaning that some sort of answers are part of the created file? Or maybe it answers the call to create the file by sending back a copy of the file, or a link to it?

Comment (george-hoffer) [reply modules/ROOT/pages/mongodb/mongodb-connector-reference.adoc]: Good catch. I removed "it." I think the idea is that the content that's saved includes answers to some inquiry.

Comment (wwarner-salesforce) [reply modules/ROOT/pages/mongodb/mongodb-connector-reference.adoc]: OK, sounds good.

File: modules/ROOT/pages/mongodb/mongodb-connector-reference.adoc [hunk 2]
```diff
@@ -14,2281 +18,2415 @@ MongoDB is an open source, high-performance, schema-free, document-oriented data
...
+* MONGO:MONGO_GRID_FS 
+* MONGO:SOCKET_OPEN 
+* MONGO:CONNECTIVITY 
+* MONGO:EXECUTION_TIMEOUT 
+* MONGO:SECURITY 
+* MONGO:MAPPING_EXCEPTION 
+* MONGO:NODE_IS_RECOVERING 
+* MONGO:INTERNAL_EXCEPTION 
+* MONGO:SERVER_ERROR 
+* MONGO:DUPLI
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/mongodb/mongodb-connector-reference.adoc]: I'm wondering if the formatting here is correct. Maybe it should be something like this?
Example: `<$set,{"key":123}>`

Comment (george-hoffer) [reply modules/ROOT/pages/mongodb/mongodb-connector-reference.adoc]: Changed line 2156 to:
For example: `<$set,{"key":123}>`

**Review by wwarner-salesforce — APPROVED** [review]

**Pull request merged**